### ISSUE TITLE
Support for ASP.NET Core RC2?

### ISSUE THREAD

**Issue by RehanSaeed** [rogeralsing/Microphone#33]
RC2 has been released and has been available for a while now. Any chance of an upgrade?

**Comment by madhon** [comment]
.Net Core is a heavliy moving target at the moment, whitness the dropping of 2 years of work on project.json in the two weeks, and then the the community annoucements 2 days ago.

Might be worth you reading [.NET Core is Boiling the Ocean](http://www.aaronstannard.com/dotnetcore-boil-ocean/) first, but i would say hold off for now until things become stable again.

**Comment by RehanSaeed** [comment]
I work on [ASP.NET MVC Boilerplate](https://github.com/RehanSaeed/ASP.NET-MVC-Boilerplate), so I am the last person to disagree about the mess that has been created but while porting to RC2, I found that the code was pretty much the same apart from a few namespace changes.

I feel that blog post is a bit over the top. The changes are all additional API's being added to the platform, the existing API's are now stable in RC2 and they will only be making changes for major bugs. It's a new platform, so you'd expect other teams at Microsoft to jump on-board towards the end and try to port their code to the platform.

The big changes are in the project, build and tooling system, which is super annoying if you work in Dev-Ops but for your average developer changing a few lines in an xproj and project.json file is not too taxing.

TLDR: The code stays largely the same, the universe around it is changing.

**Comment by Tazer** [comment]
@rogeralsing  any plans to update to RC2 ? :)

**Comment by rogeralsing** [comment]
Done :-)
Just updated the code, sadly there is no support for NancyFx provider yet as Nancy is not RC2 compat

**Comment by rogeralsing** [comment]
available on nuget as Microphone.Core 0.6.0
